### PR TITLE
Changed Leo to theLMGN, fixed my URL.

### PR DESCRIPTION
**Pull request by foxt**
Fixed some things.

http:// ==> https://www.
Leo      ==> theLMGN

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,8 +61,8 @@ <h2 class="subtitle">Who's in charge?</h2>
                         <h3>TrashyClassy</h3><span class="tag is-info">Admin</span>
                     </div>
 					 <div class="column">
-                        <a href="http://thelmgn.com/" rel="noopener noreferrer" target="_blank"><img class="someone admin" src="/img/people/leo.jpg"></a>
```
Comment (ByteOfMelon) [inline index.html:64]: Why did you remove ``noreferrer``?

**Review by ByteOfMelon — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (t5r7) [reply index.html:64]: Yeah, it's for [this](https://dev.to/ben/the-targetblank-vulnerability-by-example)

Comment (foxt) [reply index.html:64]: I was going to add a thing where it was going to show a message if you came from auroracommunity.io, but i decided against that because it's stupid.

Comment (t5r7) [reply index.html:64]: i mean the issue is probably patched by now, but ¯\\_(ツ)_/¯

Comment (ByteOfMelon) [reply index.html:64]: @theLMGN Alright, just make a new commit readding ``noreferrer`` and I'll merge

Comment (foxt) [reply index.html:64]: Done

**Review by ByteOfMelon — APPROVED** [review]
Alright, looks good
